### ISSUE TITLE
Install issues with pip (Python 3.7)

### ISSUE THREAD

**Issue by jameskeaveney** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#19]
Hi folks

I'm having an installation issue with pip. I've installed the Visual C++ 14.0 redist from microsoft, but the pip installer is still claiming that it's not installed. I'm using 64-bit python 3.7 (vanilla - not Anaconda/Enthought). Is there any workaround for this, or is this a known problem? I've copied the error output below:

Cheers

James

p.s. Not sure if it's in your control or not, but the link to the visual cpp tools site in the error message below gives me a 404 error.

```
    ....
    copying arc\arc_c_extensions.c -> build\lib.win-amd64-3.7\arc
    running build_ext
    No module named 'numpy.distutils._msvccompiler' in numpy.distutils; trying from distutils
    building 'arc.arc_c_extensions' extension
    error: Microsoft Visual C++ 14.0 is required. Get it with "Microsoft Visual C++ Build Tools": http://landinghub.visualstudio.com/visual-cpp-build-tools

    ----------------------------------------
Command "c:\users\jkeaveney\appdata\local\programs\python\python37\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\JKEAVE~1\\AppData\\Local\\Temp\\pip-install-h9d5u8oq\\ARC-Alkali-Rydberg-Calculator\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\JKEAVE~1\AppData\Local\Temp\pip-record-l2lrredq\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\JKEAVE~1\AppData\Local\Temp\pip-install-h9d5u8oq\ARC-Alkali-Rydberg-Calculator\
```

**Comment by nikolasibalic** [comment]
Hi James!

Thank you for submitting the issue. Problem was that I didn't added package wheels on PyPI for Python 3.7 (last time deployment script was changed, that Python version was not released yet). Lacking corresponding wheels, pip was trying to compile C extension on your computer, for which setting the correct environment and compilers is annoying, and shouldn't be expected from users.

I've added now (eaff7efb6b9c6fc3a45f7c9e7642c80718db3805) wheels for Python 3.7. Installation should be quick and easy. Just make sure that you are installing the latest, not cached, package by calling:

```
pip install ARC-Alkali-Rydberg-Calculator --upgrade --no-cache-dir
```

The version should be 2.0.4 (or higher). Let me know if you still have problems.

Best wishes,
Nikola
